components/brave_shields/browser/ad_block_regional_service_manager.cc

```diff
@@ -87,7 +87,7 @@ void AdBlockRegionalServiceManager::StartRegionalServices() {
     const base::Value* regional_filter_dict =
         regional_filters_dict_with_cookielist.FindDictKey(uuid);
     if (regional_filter_dict) {
-      enabled = regional_filter_dict->FindBoolKey("enabled").value_or(false);
+      enabled = regional_filter_dict->FindBoolPath("enabled").value_or(false);
```

Comment:
`FindBoolKey`

Reply:
Fixed in 2713c0a45d9d84f7cb4a310c982e48a7064bd900